### PR TITLE
[Minor update] Removed the Main build image

### PR DESCRIPTION
**Pull request by SadeekFarhan21**
The main build is linked to a file called ci-test.yml but the file doesn't exist rather now the three files are ci-test-mac.yml, ci-test-linux.yml, ci-test-windows.yml. Removed link completely for better readability but willing to choose either one of the .yml files to replace the image with the .yml file. 
image

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+p4c is a reference compiler for the P4 programming language. It supports both P4-14 and P4-16; you can find more information about P4 here and the specifications for both versions of the language here. One fact attesting to the level of quality and completeness of p4c's code is that its front-end code, mid-end code, and p4c-graphs back end are used as the basis for at least one commercially supported P4 compiler.
```
Comment (fruffy) [inline docs/index.rst:1]: I believe this was committed accidentally.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test.yml)
+[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml)
```
Comment (fruffy) [inline README.md:1]: Please add the fedora and debian links here. The debian one is the most important.

**Review by fruffy — CHANGES_REQUESTED** [review]
Good catch and thanks for your contribution. I replied with suggestions.

File: docs/index.rst (hunk 1 above)
Comment (SadeekFarhan21) [reply docs/index.rst:1]: Yes and no, I am currently working to update the documentation for the p4 compiler site. So far I was able to create a roadmap of how I would be able to create a documentation website for p4c following this https://tvm.apache.org/docs site in mind. That file was just part of the big change.

File: README.md (hunk 2 above)
Comment (SadeekFarhan21) [reply README.md:1]: I would make sure to add that. Thanks for letting me know.

File: docs/index.rst (hunk 1 above)
Comment (fruffy) [reply docs/index.rst:1]: We should add this change in a separate PR. It is good to keep your pull requests small and targeted.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test.yml)
+[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-fedora.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-fedora.yml)[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-debian.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-debian.yml) [![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml)
```
Comment (fruffy) [inline README.md:1]:
```suggestion
[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-debian.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-debian.yml)
[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-fedora.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-fedora.yml)
[![Main Build](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml/badge.svg)](https://github.com/p4lang/p4c/actions/workflows/ci-test-mac.yml)
```

File: docs/index.rst (hunk 1 above)
Comment (SadeekFarhan21) [reply docs/index.rst:1]: Great sounds good to me. I would then create a separate PR and commit there.

Comment (fruffy) [reply docs/index.rst:1]: Please do so. Until then I can not approve this PR.

**Comment by SadeekFarhan21** [comment]
I would create two different pull requests tonight both with the main build and this one

**Pull request closed without merging**